tools/topology/topology2/include/components/gain.conf

```diff
@@ -82,6 +82,20 @@ Class.Widget."gain" {
 		token_ref	"sof_tkn_comp.word"
 	}
 
+	# 'normal' = add widget name as prefix to gain name
+	# 'no_prefix' = use gain name as it is
+	DefineAttribute."name_no_prefix" {
```

Comment:
I guess this wont work in the manifest but has to become per component ?

Reply:
Oh, I forgot the old comments there. The property was firs an enum, but the I decided boolean is better. I'll remove those.

Reply:
> I guess this wont work in the manifest but has to become per component ?

Yes, but its easy to make it default behavior. This is what I was trying to accomplish all the time. Then when I started to study the manifest approach, I bumped into this way of doing it.